TechnologyAssessmentApp/TechnologyAssessmentApp/POP Pattern/MainViewController.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  MainViewController.swift
+//  TechnologyAssessmentApp
+//
+//  Created by Kshitij Godara on 24/08/18.
+//  Copyright © 2018 Nagarro. All rights reserved.
+//
+
+import UIKit
+
+class FoodImageView: UIImageView, Shakeable {
+    // other customization here
+}
+
+class MainViewController: UIViewController, ShakeDeviceProtocol {
```

Comment:
instead of creating a different VC, I will integrate it as part of the others VC. Like that when we start the app we are able to access all the features, without switching starting point of the app (storyboard/VC). A good idea will be to add it to DetailViewController